src/App.js

```diff
@@ -18,8 +18,8 @@ export default class App extends Component {
       checkedBoxes: [],
       filteredDogs: [],
     };
-    this.filterDogs = this.filterDogs.bind(this);
-    this.allDogs = this.allDogs.bind(this)
+    // this.filterDogs = this.filterDogs.bind(this);
+    // this.allDogs = this.allDogs.bind(this)
```

Comment:
Yea you can probably just delete these two lines of code. Doing this `bind` is essentially what you are doing by creating these methods as arrow functions later in this file. You will still see code that looks like this (binding in the constructor), but it's no longer needed/recommended.